### ISSUE TITLE
some minor scoring issues

### ISSUE THREAD

**Issue by nickreich** [reichlab/forecast-repository#198]
- [ ] some scores are missing season label? this is not a critical piece for COVID-19, but why is this happening?
![image](https://user-images.githubusercontent.com/1280767/81475386-d70b6380-91d9-11ea-84b7-6ffb9eecdb1e.png)

**Comment by Khoale1096** [comment]
When I first created this Zoltar project the first available timezero was 2020-03-16 so I set that date as the season start date. As we now have forecasts that have timezeros before this date, like the one you found, they couldn't find the season label. I've updated the season_start_date to be 2019-12-30, so this issue should be resolved, but the scores on the page and in the score CSVs haven't got updated since my change. I'll let you know once this issue is resolved.
